### PR TITLE
Fix tfrecord.ipynb

### PR DESCRIPTION
**Pull request by vinmuk**
In tfrecord.ipynb, fix small bug and comment.

### CODE REVIEW

File: site/ja/tutorials/load_data/tfrecord.ipynb [hunk 1]
```diff
@@ -352,7 +352,7 @@
         "このデータセットには4つの特徴量があります。\n",
         "- `False` または `True` を表す論理値。出現確率は等しいものとします。\n",
         "- ランダムなバイト値。全体において一様であるとします。\n",
-        "- `[-10000, 10000)` の範囲から一様にサンプリングした整数値。\n",
+        "- `[-0, 5)` の範囲から一様にサンプリングした整数値。\n",
```
Comment (kiszk) [inline site/ja/tutorials/load_data/tfrecord.ipynb]: [Strong suggestion]
Based on https://github.com/tensorflow/docs/blob/master/site/en/tutorials/load_data/tfrecord.ipynb, `[-0, 5)` -> `[0, 5]`

File: site/ja/tutorials/load_data/tfrecord.ipynb [hunk 2]
```diff
@@ -372,7 +372,7 @@
         "# ブール特徴量 False または True としてエンコードされている\n",
         "feature0 = np.random.choice([False, True], n_observations)\n",
         "\n",
-        "# 整数特徴量  -10000 から 10000 の間の乱数\n",
+        "# 整数特徴量  0 から 5 の間の乱数\n",
```
Comment (kiszk) [inline site/ja/tutorials/load_data/tfrecord.ipynb]: [Strong suggestion]
Based on https://github.com/tensorflow/docs/blob/master/site/en/tutorials/load_data/tfrecord.ipynb, `5` -> `4`

File: site/ja/tutorials/load_data/tfrecord.ipynb (hunk 1 above)
Comment (kiszk) [reply site/ja/tutorials/load_data/tfrecord.ipynb]: Could you please add a translation of this sentence `a string feature generated from a string table by using the integer feature as an index`?

File: site/ja/tutorials/load_data/tfrecord.ipynb [hunk 3]
```diff
@@ -352,7 +352,7 @@
         "このデータセットには4つの特徴量があります。\n",
         "- `False` または `True` を表す論理値。出現確率は等しいものとします。\n",
         "- ランダムなバイト値。全体において一様であるとします。\n",
```
Comment (kiszk) [inline site/ja/tutorials/load_data/tfrecord.ipynb]: Was this line dropped from [the original file](https://github.com/tensorflow/docs/blob/master/site/en/tutorials/load_data/tfrecord.ipynb) now?

**Comment by kiszk** [comment]
Thank you for submitting a PR. Could you address the comments?

**Comment by vinmuk** [comment]
Thank you for your comments.
Reflect your comments.

File: site/ja/tutorials/load_data/tfrecord.ipynb (hunk 3 above)
Comment (vinmuk) [reply site/ja/tutorials/load_data/tfrecord.ipynb]: `"- ランダムなバイト値。全体において一様であるとします。\n",`
is deleted, so I delete it in reflect comments.

File: site/ja/tutorials/load_data/tfrecord.ipynb (hunk 1 above)
Comment (vinmuk) [reply site/ja/tutorials/load_data/tfrecord.ipynb]: Thank you for your comments.
I think `[0, 5]` should be `[0, 5)`  since original one is wrong.

**Comment by lamberta** [comment]
What's the status of this translation fix ... ready to merge?
If you find an error in the en/ version, please send a fix for that :)

**Comment by vinmuk** [comment]
Modified based on comments and waiting for re-review.

**Comment by lamberta** [comment]
Thank you.

Community translations are now located in the [tensorflow/docs-l10n](https://github.com/tensorflow/docs-l10n) repo (see [this message](https://groups.google.com/a/tensorflow.org/g/docs/c/GJPq7S_fw4c/m/lA1qfBX2EQAJ) for context).

I'll move over/duplicate existing pull requests as I find time. In the meantime, please feel free to create a new pull request in that repo 😀 Then you can close this PR and add a link to the new one.

**Pull request closed without merging**